### DISCUSSION TITLE
Serilog message body

### DISCUSSION THREAD

**Discussion by Depechie** [serilog-contrib/serilog-sinks-grafana-loki#137] [Q&A]
Was wondering if we are doing something wrong, or misjudging the features?
Currently we use Serilog to format our log entries and use the loki sink to push it in json format.

If we look at a result in Loki we get this

<img width="744" alt="Screenshot 2022-10-07 155644" src="https://user-images.githubusercontent.com/351693/194571452-9632c6b4-c10d-4b4b-8862-9b10eb70c0c5.png">

If we now use the propertiesAsLabel feature we get following result

<img width="746" alt="Screenshot 2022-10-07 160254" src="https://user-images.githubusercontent.com/351693/194572236-1a7283f4-ee32-490d-9e2f-62685ad78403.png">

So the actual properties are indeed transformed, but that seems to also impact the payload structure... the Message line now no longer maps the data but only show the placeholders, like {RequestPath}.

Is this expected behaviour?

**Comment by mishamyte** [comment]
Hi @Depechie,

`propertiesAsLabel` feature only maps event properties to a static labels.
Be careful of it, because if your properties, mapped to labels will has a lot of unique value, you could face [high cardinality problem](https://grafana.com/blog/2020/04/21/how-labels-in-loki-can-make-log-queries-faster-and-easier/#cardinality)
[This article could be useful](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Mapping-properties-to-labels).

And this behavior doesn't looks like an expected. Thank you, will do an investigation

**Reply by mishamyte** [reply]
I can confirm that it is a bug, so I extracted it to #138. 

Will look for the way how to solve it gracefully

**Reply by Depechie** [reply]
Thx!
